.travis.yml

```diff
@@ -141,7 +141,7 @@ install:
 - conda config --set always_yes yes --set changeps1 no
 - conda update -q conda
 - conda info -a
-- conda create -q -n p4env python=$PYTHON_VER numpy cmake ci-psi4-lt psi4-lt-mp dftd3 gcp -c psi4/label/dev -c psi4
+- conda create -q -n p4env python=$PYTHON_VER numpy cmake ci-psi4-lt psi4-lt-mp mpmath dftd3 gcp -c psi4/label/dev -c psi4
```

Comment:
Do you want me to drop this dependency? Its fairly common, but the data required for it can be computed once and saved.

Reply:
unless it causes problems I'm not expecting, `mpmath` (and `numpy`) are fine. good to know that it could be removed, though.